recipe-client-addon/bootstrap.js

```diff
@@ -68,7 +70,13 @@ this.startup = function() {
   CleanupManager.addCleanupHandler(
     () => Preferences.ignore(PREF_LOGGING_LEVEL, LogManager.configure));
 
-  RecipeRunner.init();
+  // Initialize experiments first to avoid a race between initializing prefs
+  // and recipes rolling back pref changes when experiments end.
+  PreferenceExperiments.init().catch(err => {
+    log.error("Failed to initialize preference experiments:", err);
```

Comment:
Yeah, I don't know how to do that either. I guess you could write a browser test that disables and re-enables the add-on (using add-on manager APIs)? I think that'd run this code, maybe?